src/chainstate/burn/db/burndb.rs

```diff
@@ -139,6 +144,11 @@ impl FromRow<BlockSnapshot> for BlockSnapshot {
         let canonical_stacks_tip_hash = BlockHeaderHash::from_column(row, "canonical_stacks_tip_hash")?;
         let canonical_stacks_tip_burn_hash = BurnchainHeaderHash::from_column(row, "canonical_stacks_tip_burn_hash")?;
 
+
+        //
```

Comment:
missing a comment? ;)

Reply:
I attempted to reconstruct my mind's intended comment there: fd819305f